mne/viz/ica.py

```diff
@@ -279,10 +279,15 @@ def plot_ica_properties(ica, inst, picks=None, axes=None, dB=True,
     dB: bool
         Whether to plot spectrum in dB. Defaults to True.
     plot_std: bool | float
-        Whether to plot standard deviation in ERP/ERF and spectrum plots.
-        Defaults to True, which plots one standard deviation above/below.
-        If set to float allows to control how many standard deviations are
-        plotted. For example 2.5 will plot 2.5 standard deviation above/below.
+        Whether to plot standard deviation/confidence intervals in ERP/ERF and
+        spectrum plots.
+        Defaults to True, which plots one standard deviation above/below for
+        the spectrum. If set to float allows to control how many standard
+        deviations are plotted for the spectrum. For example 2.5 will plot 2.5
+        standard deviation above/below.
+        For the ERP/ERF, by default, plot the 95 percent parametric confidence
+        interval is calculated. To change this, use ``ci`` in ``ts_args`` in
+        ``image_args`` (see below).
```

Comment:
why is this called ci at some place and std at others? When I read plot_std for me you report +/- 1 standard deviation on the plots. This needs to be made consistent.

Reply:
Yes, for psd plot we first used +/-1 std but later when evoked plot was refactored to use plot_compare_evokeds it used 95% CI. So currently spectrum uses +/- 1 std and evoked plot uses CI.

Reply:
I am not sure what would be best. The evoked reuses `plot_compare_evokeds`. We could use the one-sided CI functionality in both evoked and psd (and add it to `mne.stats._ci´), and call it a 95% CI for both.

Reply:
@mmagnuski ?